### ISSUE TITLE
[Bug]: Linux Update Error

### ISSUE THREAD

**Issue by ThioJoe** [ThioJoe/YT-Spammer-Purge#689]
```
Getting info about latest updates...
------------------------------------------------------------------------------------------
 A new version is available! Visit TJoe.io/latest to see what's new.
   > Current Version: 2.15.1
   > Latest Version: 2.16.2
------------------------------------------------------------------------------------------

Update Now? (y/n): y

> Downloading version: 2.16.2

> Extracting...
> Installing...
Traceback (most recent call last):
  File "/usr/lib/python3.8/shutil.py", line 791, in move
    os.rename(src, real_dst)
FileNotFoundError: [Errno 2] No such file or directory: './config' -> '/home/ocean/YT-Spammer-Purge-2.15.1/temp/YT-Spammer-Purge-2.16.2/config'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "YTSpammerPurge.py", line 1611, in <module>
    main()
  File "YTSpammerPurge.py", line 1588, in main
    continueRunning = primaryInstance(miscData)
  File "YTSpammerPurge.py", line 875, in primaryInstance
    files.check_for_update(version, updateReleaseChannel)
  File "/home/ocean/YT-Spammer-Purge-2.15.1/Scripts/files.py", line 350, in check_for_update
    move("./config", extraFolderPath)
  File "/usr/lib/python3.8/shutil.py", line 811, in move
    copy_function(src, real_dst)
  File "/usr/lib/python3.8/shutil.py", line 435, in copy2
    copyfile(src, dst, follow_symlinks=follow_symlinks)
  File "/usr/lib/python3.8/shutil.py", line 264, in copyfile
    with open(src, 'rb') as fsrc, open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: './config'
------------------------------------------------
Error Message: [Errno 2] No such file or directory: './config'

Unknown Error - Code: Z-3 occurred. If this keeps happening, consider posting a bug report
on the GitHub issues page, and include the above error info.
Short Link: TJoe.io/bug-report


_Originally posted by @Lagicrus in https://github.com/ThioJoe/YT-Spammer-Purge/discussions/684#discussioncomment-2216170_
```

Problem appears to not be related to config files, but the linux update function. Which calls the folder that other stuff gets put into "config", though it has nothing to do with the config files.

**Comment by compsup** [comment]
This problem was fixed in #620. You are running an old version, you will need to manually update too the newest version and then it should work from there.

**Comment by compsup** [comment]
@ThioJoe This can be closed.
